packages/auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/auth",
-  "version": "3.6.1",
+  "version": "3.7.0",
   "engines": {
```

Comment:
Maybe it makes sense to remove `@standardnotes/features` [dependency](https://github.com/standardnotes/snjs/blob/master/packages/auth/package.json#L29) at all, since we won't be using it with these changes anymore?

Reply:
hmm good point

Reply:
Thanks @vardan-arm! 🙂